src/stactools/noaa_c_cap/constants.py

```diff
@@ -47,6 +47,12 @@
         title="C-CAP Classification Scheme and Class Definitions",
         media_type="application/pdf",
     ),
+    Link(
+        rel="license",
+        target="https://coast.noaa.gov/digitalcoast/data/ccapregional.html",
+        title="C-CAP Regional Land Cover and Change",
+        media_type="text/html",
```

Comment:
Seems weird that pystac doesn't have an HTML media type.

Reply:
Yeah I was surprised by that as well.